winui/Kanban/cards.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: Cards in WinUI Kanban control | Syncfusion
+description: This section describes about cards in Syncfusion WinUI Kanban (SfKanban) control, its elements and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Cards in WinUI Kanban Control
+
+Cards in the WinUI Kanban Control provide a visual representation of tasks as they move through different stages of a workflow. Each card corresponds to a task and displays key information, such as the title, description, tags, and status, helping users track progress and manage tasks effectively.
```

Comment:
The Kanban cards visually represent tasks and their progression through various stages. The default UI of each card can be customized using the following properties of the KanbanModel.

Reply:
Updated.